ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -398,6 +401,33 @@ - (NSDictionary *)powerState
     resolve(@(locationServicesEnabled));
 }
 
+RCT_EXPORT_METHOD(getUserAgent:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
```

Comment:
where did you get this function from and what do you use it for?

Reply:
I got it from a PR from the official package
https://github.com/react-native-community/react-native-device-info/pull/757/files